src/He.PipelineAssessment.UI/Features/Workflow/LoadConfirmationScreen/LoadConfirmationScreenRequestHandler.cs

```diff
@@ -62,16 +62,19 @@ public LoadConfirmationScreenRequestHandler(IElsaServerHttpClient elsaServerHttp
 
 
                             if (!string.IsNullOrEmpty(response.Data.NextWorkflowDefinitionIds) && !currentAssessmentToolWorkflowInstance.IsVariation)
-                            {
+                            { 
                                 var nextWorkflows = new List<AssessmentToolInstanceNextWorkflow>();
                                 var workflowDefinitionIds = response.Data.NextWorkflowDefinitionIds.Split(',', StringSplitOptions.TrimEntries);
                                 foreach (var workflowDefinitionId in workflowDefinitionIds)
                                 {
                                     var nextWorkflow =
                                         await _assessmentRepository.GetAssessmentToolInstanceNextWorkflow(currentAssessmentToolWorkflowInstance.Id,
                                             workflowDefinitionId);
+                                    var allAssessmentToolWorkflowInstances = await _assessmentRepository.GetAssessmentToolWorkflowInstances(currentAssessmentToolWorkflowInstance.AssessmentId);
```

Comment:
Just checking - does this get assessments that have been rolled back/overridden?  Could give a false-positive if so.